tests/src/test_common.h

```diff
@@ -104,8 +104,15 @@ THE SOFTWARE.
 #define popen(x,y) _popen(x,y)
 #define pclose(x) _pclose(x)
 #define setenv(x,y,z) _putenv_s(x,y)
+#define unsetenv _putenv
+#define PLATFORM_NAME "WINDOWS"
+#define HIP_VISIBLE_DEVICES "HIP_VISIBLE_DEVICES="
+#define CUDA_VISIBLE_DEVICES "CUDA_VISIBLE_DEVICES="
 #else
 #define aligned_free(x) free(x)
+#define PLATFORM_NAME "OTHER"
+#define HIP_VISIBLE_DEVICES "HIP_VISIBLE_DEVICES"
+#define CUDA_VISIBLE_DEVICES "CUDA_VISIBLE_DEVICES"
```

Comment:
1. Remove redefine: `#define PLATFORM_NAME "..."`, and use convenient `#ifdef _WIN64 ... #else ... #endif`.
2. Use `extern const char* HIP_VISIBLE_DEVICES;` and `const char* HIP_VISIBLE_DEVICES="HIP_VISIBLE_DEVICES"`; instead of defines. Btw, not `std::string` type, cause the header is C.
3. I'd rename` HIP(CUDA)_VISIBLE_DEVICES` to `HIP(CUDA)_VISIBLE_DEVICES_STR` to avoid possible collisions.

Reply:
I'm not clear on 1. and 2. in your feedback.
For 1. the defines are to have the conditional statements in hipEnvVarDriver.cpp know which OS is being used. I'm not aware on how to tell with OS it is with just _WIN64. 
For 2. By putting extern const char* HIP_VISIBLE_DEVICES; in test_common.h and then defining const char* HIP_VISIBLE_DEVICES="HIP_VISIBLE_DEVICES"; in hipEnvVarDriver.cpp, When do I declare it to be  "HIP_VISIBLE_DEVICES=" vs "HIP_VISIBLE_DEVICES"? How can I tell which one to use?